### PR TITLE
Editorial updates - fix errors and improve content

### PR DESCRIPTION
**Pull request by SueChaplain**
Three main changes:

1. Default value for -Xmso on 64-bit z/OS was incorrect.
(#194)

2. Compiler levels for JDK 8 Linux are changed. (#191)

3. New graphic available for gencon policy. Added with
minor edits to the surrounding text.

Closes: #194
Closes: #191

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
Not sure how that happened. Leaked some changes from another branch I'm working on. I'll fix Monday.

File: docs/openj9_support.md [hunk 1]
```diff
@@ -162,16 +162,13 @@ The project build and test OpenJDK with OpenJ9 on a number of platforms. The ope
 
 | Platform                                    | Operating system          |  Compiler                       |
 |---------------------------------------------|---------------------------|---------------------------------|
-| Linux x86 64-bit                            | Ubuntu 16.04              | gcc 4.8.5                       |
-| Linux on POWER&reg; LE 64-bit               | Ubuntu 16.04              | gcc 4.8.5                       |
-| Linux on IBM Z&reg; 64-bit                  | Ubuntu 16.04              | gcc 4.8.5                       |
```
Comment (pshipton) [inline docs/openj9_support.md]: zLinux should be 7.4

Comment (pshipton) [reply docs/openj9_support.md]: The OpenJ9 build machines have been updated, although the issue may not reflect this yet.

Comment (SueChaplain) [reply docs/openj9_support.md]: - I've changed zlinux to 7.4. When I looked last week the issue was still open, hence updated to 7.3.

- cgroup changes are removed and will go into a separate PR once I get the go-ahead.

@pshipton  Are you ok to unblock for merging please?

**Pull request merged**